curricula/models.py

```diff
@@ -71,10 +74,24 @@ def internationalizable_fields(cls):
     def __unicode__(self):
         return self.title
 
+    def can_access(self, request):
+        return request.user.has_perm('curricula.access_all_curricula') or request.user.id == self.user_id
+
+    def can_add(self, request):
+        return self.can_access(request)
+
+    def can_change(self, request):
+        return self.can_access(request)
+
+    def can_delete(self, request):
+        return self.can_access(request)
+
     def can_move(self, request, new_parent):
         if new_parent is not None:
             msg = 'Curriculum must be a top level object'
             raise PageMoveException(msg)
+        if not request.user.has_perm('curricula.access_all_curricula'):
```

Comment:
is there a reason this isn't using `can_access`?

Reply:
Yes! We do not want the partner to be able to move their `Curriculum` around within admin Page view shown in the anigifs, even if they own it. I've added a comment.